drivers/SmartThings/zwave-switch/src/multichannel-device/init.lua

```diff
@@ -0,0 +1,93 @@
+-- Copyright 2022 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+local cc = require "st.zwave.CommandClass"
+local capabilities = require "st.capabilities"
+local st_device = require "st.device"
+local MultiChannel = (require "st.zwave.CommandClass.MultiChannel")({ version = 3 })
+
+local map_device_class_to_profile = {
+  [0x10] = "metering-switch",
```

Comment:
Switch binary device type maps to metering-switch? Im guessing this is just an assumption to better support all devices because they dont report functionality well via Z-Wave protocol?

Reply:
It is mapped to metering switch because [Switch Endpoint](https://github.com/SmartThingsCommunity/SmartThingsPrivate/blob/master/devicetypes/smartthings/switch-endpoint.src/switch-endpoint.groovy) also contains power and energy capabilities